packages/babel-types/src/asserts/generated/index.js

```diff
@@ -224,12 +224,12 @@ export function assertArrowFunctionExpression(
 export function assertClassBody(node: Object, opts?: Object = {}): void {
   assert("ClassBody", node, opts);
 }
-export function assertClassDeclaration(node: Object, opts?: Object = {}): void {
-  assert("ClassDeclaration", node, opts);
-}
 export function assertClassExpression(node: Object, opts?: Object = {}): void {
   assert("ClassExpression", node, opts);
 }
+export function assertClassDeclaration(node: Object, opts?: Object = {}): void {
```

Comment:
I think they are sorted alphabetically?

Reply:
This file is automatically generated. I had to swap then in the `definitions/es2015` file, so they got swapped here.